packages/snaps-utils/package.json

```diff
@@ -90,6 +90,7 @@
     "@metamask/utils": "^10.0.0",
     "@noble/hashes": "^1.3.1",
     "@scure/base": "^1.1.1",
+    "async-mutex": "^0.4.0",
```

Comment:
Seems that we use 0.5.0 in the extension dep-tree already (e.g. in the KeyringController), could we do the same here?